packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.tsx

```diff
@@ -188,10 +199,8 @@ export class GuxCalendar {
         };
       }
 
-      // Disable a date that is outside the defined date range boundaries
-      const disabled =
-        (this.minValue && currentDate.getTime() <= this.minValue.getTime()) ||
-        (this.maxValue && currentDate.getTime() > this.maxValue.getTime());
+      // Check if a date is outside the defined date range boundaries
+      const valid = this.isValidDate(currentDate);
```

Comment:
Do not make this change from `disabled` to `valid` it makes the code harder to follow below for no benefit
Use something like
```
const disabled = !this.isValidDate(currentDate);
```
or
```
const disabled = this.isInvalidDate(currentDate);
```